lib/yard/handlers/base.rb

```diff
@@ -181,7 +181,7 @@ def inherited(subclass)
         # the handlers, otherwise the same code will be parsed
         # multiple times and slow YARD down.
         #
-        # @param [Parser::RubyToken, Symbol, String, Regexp] matches
+        # @param [Parser::Ruby::Legacy::RubyToken, Symbol, String, Regexp] matches
```

Comment:
I'm not 100% certain this is the right one, but it's the only `RubyToken` I could find in this gem.